### PR TITLE
Adding initiateExchange EP to client lib

### PR DESCRIPTION
**Pull request by brandonrobinson5060**
Client lib didn't have initiateExchange endpoint on api v2.  Adding it here

### CODE REVIEW

**Comment by lylepratt** [comment]
Would be good if you guys could merge this! We're using your Node client, but it does not have support for the exchange API so @brandonrobinson5060 added it.

File: submodules/client.js [hunk 1]
```diff
@@ -57,6 +57,22 @@ class OpenNodeClient {
...
   }
 
+  async initiateExchange(exchange) {
+   try {
+     let new_instance = axios.create();
+     new_instance.defaults.baseURL = (this.env === 'live') ? 'https://api.opennode.com/v2' : 'https://dev-api.opennode.com/v2';
+     new_instance.defaults.timeout = 15000;
+     new_instance.defaults.headers = { 'Authorization': this.api_key, 'user_agent': version };
+
+     const response = await new_instance.post('/exchanges', exchange);
```
Comment (johnnyasantoss) [inline submodules/client.js:67]: Is there a reason to create a new axios instance over `this.instance`?

**Review by johnnyasantoss — CHANGES_REQUESTED** [review]
Looks good, just added a question on reusing the current axios client.

File: submodules/client.js (hunk 1 above)
Comment (johnnyasantoss) [reply submodules/client.js:67]: Ah, nevermind, just realized this is for `v2` and `this.instance` is for `v1`

Comment (brandonrobinson5060) [reply submodules/client.js:67]: hey @johnnyasantoss - 

I believe "this.instance" refers to the v1 api URL whereas the initiateExchange ep is only on v2. 

Similar to how "initiateWithdrawalAsync"  is setup 

"async initiateWithdrawalAsync(withdrawal) {
    try {
      let new_instance = axios.create();
      new_instance.defaults.baseURL = (this.env === 'live') ? 'https://api.opennode.com/v2' : 'https://dev-api.opennode.com/v2';
      new_instance.defaults.timeout = 15000;
      new_instance.defaults.headers = { 'Authorization': this.api_key, 'user_agent': version };

      const response = await new_instance.post('/withdrawals', withdrawal);
      return response.data.data;
    }
    catch (error) {
      throw Exception(error.response.status, error.response.statusText, error.response.data.message);
    }
  }"

**Review by johnnyasantoss — APPROVED** [review]
LGTM

**Comment by brandonrobinson5060** [comment]
Hey there @johnnyasantoss - hope all is well. Not sure how often you guys merge/deploy but was just wondering if we could get this merged soon. Thanks!

**Comment by johnnyasantoss** [comment]
> Hey there @johnnyasantoss - hope all is well. Not sure how often you guys merge/deploy but was just wondering if we could get this merged soon. Thanks!

It's published as `1.4.0`. Thank you for your contribution!

**Pull request merged**